samples/drivers/led/pwm/sample.yaml

```diff
@@ -17,6 +17,6 @@ tests:
         - "Turned on"
         - "Turned off"
         - "Increasing brightness gradually"
-        - "Blinking on: 0.1 sec, off: 0.1 sec"
-        - "(Blinking on: 1 sec, off: 1 sec|Cycle period not supported)"
+        - "Blinking on: ([0-9]+) msec, off: ([0-9]+) msec"
```

Comment:
maybe I'm missing something but I don't see how this actually fixes #77970? 

The primary failure on the blinky stage of the test is that the value passed to the driver is too large for a 16 bit register.

Reply:
@dleach02 you are right regarding NXP boards. This PR is just the first step to pass the test, because the test fails now because of the change of log output for all boards from all vendors,. Based on the log provided by @nordic-piks in the https://github.com/zephyrproject-rtos/zephyr/issues/77970, the PR fixes the nrf case.

For NXP boards mentioned in https://github.com/zephyrproject-rtos/zephyr/issues/77970 another fix is needed.

Removed the link to #77970.